tascrum/serializers.py

```diff
@@ -565,8 +602,8 @@ class Meta:
         fields = ['id', 'lists']
 
     def get_lists(self, obj):
-        lists = obj.lboard.all()
-        return ListBoardTimelineSerializer(lists, many=True).data
+        lists = obj.lboard.filter(board=obj.id)
```

Comment:
why did you add filter instead of all() function

Reply:
because we only want the board objects specified in the url.